toolkit/scripts/build_tag_imagecustomizer.mk

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Azure Linux Image Customizer version.
+# This is using semantic versioning.
+#
+# IMAGE_CUSTOMIZER_VERSION should have the format:
+#
+#   <major>.<minor>.<patch>
+#
+# and should hold the value of the next (or current) official release, not the previous official
+# release.
+IMAGE_CUSTOMIZER_VERSION ?= 0.3.0
+IMAGE_CUSTOMIZER_VERSION_PREVIEW ?= -dev.$(DATETIME_AS_VERSION)+$(GIT_COMMIT_ID)
+IMAGE_CUSTOMIZER_FULL_VERSION := $(IMAGE_CUSTOMIZER_VERSION)$(IMAGE_CUSTOMIZER_VERSION_PREVIEW)
```

Comment:
Since we're here: as convention we try to reserve capital letter variables for input arguments to the tools. Is it intentional to make the customizer's version modifiable? If no, can you switch to lower case names? If yes, can you extend the documentation?

Reply:
Fixed.